### PR TITLE
test: remove unused src_dir param from run_tests after CMake migration

### PR DESCRIPTION
**Pull request by l0rinc**
The `src_dir` usage was removed in  https://github.com/bitcoin/bitcoin/commit/a8a2e364acf55bbe18404ab21f852d52257bcb6d#diff-437d7f6e9f2229879b60aae574a8217f14c643bbf3cfa9225d8011d6d52df00cL598, making the parameter unused.

### CODE REVIEW

**Comment by l0rinc** [comment]
Done, thank @maflcko

File: test/functional/test_runner.py [hunk 1]
```diff
@@ -595,7 +594,7 @@ def run_tests(*, test_list, src_dir, build_dir, tmpdir, jobs=1, enable_coverage=
         print(f"{BOLD[1]}WARNING!{BOLD[0]} There may be insufficient free space in {tmpdir} to run the Bitcoin functional test suite. "
               f"Running the test suite with fewer than {min_space // (1024 * 1024)} MB of free space might cause tests to fail.")
 
-    tests_dir = build_dir + '/test/functional/'
+    tests_dir = f"{build_dir}/test/functional/"
```
Comment (hebasto) [inline test/functional/test_runner.py:597]: nit: This change seems irrelevant.

**Review by hebasto — APPROVED** [review]
ACK 76a28306f1e2c0828590eded0bf6b178f6b889da.

File: test/functional/test_runner.py (hunk 1 above)
Comment (l0rinc) [reply test/functional/test_runner.py:597]: I put it to a separate commit originally, @maflcko suggested I merge it into the same commit.
If you have strong feelings about it, I'll revert - but seems more idiomatic Python this way.

**Comment by davidgumberg** [comment]
ACK https://github.com/bitcoin/bitcoin/commit/76a28306f1e2c0828590eded0bf6b178f6b889da

**Comment by fanquake** [comment]
@davidgumberg fyi you've ack'd the previous commit has here.
